actnow/petitions/models.py

```diff
@@ -63,6 +63,9 @@ class Signature(TimestampedModelMixin):
         max_length=255,
         blank=True,
     )
+    sign_anonymously = models.BooleanField(
```

Comment:
```suggestion
    anonymous = models.BooleanField(
```

Signature means it has already been signed so it's more of was it anonymously signed?